static/js/app/campaign_results.js

```diff
@@ -2,6 +2,20 @@ var map = null
 
 // statuses is a helper map to point result statuses to ui classes
 var statuses = {
+    "HTML Opened": {
+        slice: "ct-slice-donut-opened",
+        legend: "ct-legend-opened",
+        label: "label-danger",
+        icon: "fa-file",
+        point: "ct-point-opened"
+    },
+    "Doc Opened": {
```

Comment:
"Doc" is pretty informal - we'll want to change this to "attachment" or something more official sounding.